stl/inc/memory

```diff
@@ -2362,6 +2385,12 @@ void _Uninitialized_fill_multidimensional_n(_Ty* const _Out, const size_t _Size,
     } else if constexpr (_Fill_memset_is_safe<_Ty*, _Ty>) {
         _Fill_memset(_Out, _Val, _Size);
     } else {
+        if constexpr (_Fill_zero_memset_is_safe<_Ty*, _Ty>) {
+            if (_Is_scalar_zero(_Val)) {

```

Comment:
Ditto "Does this new special case have test coverage?"

Reply:
AFAIK this function is not tested anywhere so I'm unsure what to do

Reply:
`_Uninitialized_fill_multidimensional_n` and `_Uninitialized_fill_multidimensional_n_al` are both used to implement `make_shared` for arrays: I assume we can add test cases to `tests/std/tests/P0674R1_make_shared_for_arrays` that will result in calls to these functions.

Reply:
Done